Tests/MapboxNavigationTests/InstructionsBannerViewSnapshotTests.swift

```diff
@@ -416,13 +413,20 @@ class InstructionsBannerViewSnapshotTests: InstructionBannerTest {
             override func apply() {
                 super.apply()
                 
-                PrimaryLabel.appearance(whenContainedInInstancesOf: [InstructionsBannerView.self]).normalTextColor = UIColor.green
-                GenericRouteShield.appearance().highlightColor = UIColor.blue
-                ExitView.appearance().highlightColor = UIColor.yellow
+                let traitCollection = UITraitCollection(userInterfaceIdiom: .phone)
+                // Check that PrimaryLabel.normalTextColor is not used
+                PrimaryLabel.appearance(for: traitCollection).normalTextColor = UIColor.blue
+                PrimaryLabel.appearance(for: traitCollection, whenContainedInInstancesOf: [InstructionsBannerView.self]).textColorHighlighted = UIColor.green
```

Comment:
Don't forget to remove the `whenContainedInInstancesOf:` as well.

Reply:
ah missed one. thanks for catching it!